src/store/store.ts

```diff
@@ -1197,6 +1185,47 @@ export const useStore = defineStore('store', {
 
     getIsRestorationTypeCourtOrder (): boolean {
       return !!this.stateModel.restoration.courtOrder?.fileNumber
+    },
+
+    /** The special resolution object. */
+    getSpecialResolution (): SpecialResolutionIF {
```

Comment:
Not sure how you'd like to order these? I put the latest at the bottom for now

Reply:
No decision has been made on this. Sometimes related methods are together; other times it's alphabetical. What do you think?

Reply:
alphabetical would probably be good.

Reply:
Yup putting down note to create ticket

Reply:
New ticket is bcgov/entity#16489.